### DISCUSSION TITLE
Can libjpeg62-turbo replace libjpeg ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [python-pillow/Pillow#7189] [Q&A] [answered]
Hi,

on the [installation](https://pillow.readthedocs.io/en/stable/installation.html#building-from-source) page we can see that `libjpeg` & `zlib` are required...

I see on my system that I have already `libjpeg62-turbo` could this be enough / replace `libjpeg` to install pillow ?

Thanks.

**Comment by radarhere** [comment, marked as answer]
Sure. When we test Debian 11, we use libjpeg62-turbo-dev - https://github.com/python-pillow/docker-images/blob/46a3e650ac3695f72d7ff522211d100fb95e898d/debian-11-bullseye-x86/Dockerfile#L44

**Reply by radarhere** [reply]
@BobSquarePants did that answer your question?

**Reply by BobSquarePants** [reply]
Yes thanks @radarhere (sorry I can take time to react sometimes but I always comeback :) )
